src/components/DropDown.js

```diff
@@ -20,14 +20,9 @@ class Dropdown extends Component {
     };
   }
 
-  shouldComponentUpdate(nextProps) {
-    console.log('DROP DOWN next props', nextProps);
-    return false;
-  }
-
   _renderOptionItems() {
     return this.props.items.map((item, key) => {
-      return <Option key={key} value={item.text}> {item.text} </Option>;
+      return <Option key={key} value={item.name}> {item.name} </Option>;
```

Comment:
renderOptionItems is mapping over the collections in answerOptions, so the value should be back to text.